kuma/actioncounters/migrations/0003_update_unique_hashes.py

```diff
@@ -16,7 +16,7 @@ def forwards(self, orm):
         # but wasn't implemented before this migration. Doing it now, so the
         # data migration has a smaller set to work with.
         (orm.ActionCounterUnique.objects
-            .filter(user=None, modified__lt=datetime.datetime(2011, 8, 1))
+            .filter(user=None, modified__lt=timezone.datetime(2011, 8, 1))
```

Comment:
No need to use the datetime object from the timezone library here, use the old one instead.